addons/spreadsheet_dashboard/security/security.xml

```diff
@@ -6,15 +6,15 @@
         <field name="domain_force">[('group_ids', 'in', user.groups_id.ids)]</field>
     </record>
 
-    <record model="ir.module.category" id="spreadsheet_dashboard.dashboard_management">
+    <record id="base.module_category_productivity_dashboard" model="ir.module.category">
```

Comment:
I think this requires a migration script with a call to "rename_xmlid" to properly update references.